domains/hafi.json

```diff
@@ -0,0 +1,12 @@
+{
+	"owner": {
+		"username": "HafiTheCat",
+		"email": "",
+		"discord": "hafi_the_cat"
+	},
+	"description": "Website about about me",
+	"repo": "https://github.com/HafiTheCat/hafi_the_cat.github.io",
+	"record": {
+		"CNAME": "hafithecat.github.io/hafi_the_cat.github.io"
```

Comment:
CNAME cant have anything after / (remove / and the stuff after it and then ill merge it)